step.yml

```diff
@@ -0,0 +1,46 @@
+title: Flank
+summary: "Run your tests using Flank."
+description: "Run your tests using Flank. The step will automaticall detect which project type your flank config uses and the corresponding flank command fille be run."
```

Comment:
The step will __automatically__ detect which project type your flank config uses and the corresponding flank command __will__ be run.